README.md

```diff
@@ -60,29 +62,53 @@ module parameter to your service's entry point.
 
 Finally, **start your service with `npm start`**. In npm >= 2.0 (node 0.12 or iojs), you can also pass parameters to `service-runner` like this: `npm start -- -c /etc/yourservice/config.yaml`.
 
-For node 0.10 support, you can create a small wrapper script like this:
+### As a library
+
+Service-runner can also be used to run services within an application. This is
+useful for node 0.10 support, but can also be used to run services for testing
+or other purposes.
+
+Example script for **starting** a service, using commandline options:
+
 ```javascript
 var ServiceRunner = require('service-runner');
 new ServiceRunner().run();
 ```
+It is also possible to skip commandline options, and pass in a config
+directly to `ServiceRunner.run()` (see [the config section](#config_loading)
+for details on available options). Here is an example demonstrating this, as
+well as return values & the `stop()` method:
 
-All file paths in the config are relative to the application base path. 
-The base path is an absolute path to the folder where your application 
-is located (where `package.json` file is located).
-
-By default, we assume that your project depends on `service-runner` and 
-you follow standard node project layout. However, if a custom layout is 
-used, you must override the app base path with either:
-- `APP_BASE_PATH` environment variable
-- `app_base_path` config stanza.
-
-We are also working on a [standard
-template](https://github.com/wikimedia/service-template-node) for node
-services, which will set up this & other things for you.
+```javascript
+var ServiceRunner = require('service-runner');
+var runner = new ServiceRunner();
```

Comment:
I found that you really need to pass in `new ServiceRunner({ num_workers: -1 })` here, otherwise yargs tries to set options from argv, which isn't expected in a library.